pkg/api/util/matrix/matrix_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package quota
+package matrix
```

Comment:
Why are you calling this package matrix? It doesn't seem to have anything to do with matrices?